tests/e2e/connect/info_test.go

```diff
@@ -0,0 +1,244 @@
+	_ context.Context,
+	_ *connect.Request[infov1.GetNodeVersionRequest],
+) (*connect.Response[infov1.GetNodeVersionResponse], error) {
+	return connect.NewResponse(&infov1.GetNodeVersionResponse{
+		Version: "stub-v1.2.3",
+	}), nil
+}
+
+func (*stubInfoService) GetNodeID(
+	_ context.Context,
+	_ *connect.Request[infov1.GetNodeIDRequest],
+) (*connect.Response[infov1.GetNodeIDResponse], error) {
+	return connect.NewResponse(&infov1.GetNodeIDResponse{
+		NodeId: "NodeID-stub-node",
+	}), nil
+}
+
+func (*stubInfoService) GetNodeIP(
+	_ context.Context,
+	_ *connect.Request[infov1.GetNodeIPRequest],
+) (*connect.Response[infov1.GetNodeIPResponse], error) {
+	return connect.NewResponse(&infov1.GetNodeIPResponse{
+		Ip: "127.0.0.1",
+	}), nil
+}
+
+func (*stubInfoService) GetNetworkID(
+	_ context.Context,
+	_ *connect.Request[infov1.GetNetworkIDRequest],
+) (*connect.Response[infov1.GetNetworkIDResponse], error) {
+	return connect.NewResponse(&infov1.GetNetworkIDResponse{
+		NetworkId: 1,
+	}), nil
+}
+
+func (*stubInfoService) GetNetworkName(
+	_ context.Context,
+	_ *connect.Request[infov1.GetNetworkNameRequest],
+) (*connect.Response[infov1.GetNetworkNameResponse], error) {
+	return connect.NewResponse(&infov1.GetNetworkNameResponse{
+		NetworkName: "network",
+	}), nil
+}
+
+func (*stubInfoService) GetBlockchainID(
+	_ context.Context,
+	_ *connect.Request[infov1.GetBlockchainIDRequest],
+) (*connect.Response[infov1.GetBlockchainIDResponse], error) {
+	return connect.NewResponse(&infov1.GetBlockchainIDResponse{
+		BlockchainId: "blockchain",
+	}), nil
+}
+
+func (*stubInfoService) Peers(
+	_ context.Context,
+	_ *connect.Request[infov1.PeersRequest],
+) (*connect.Response[infov1.PeersResponse], error) {
+	return connect.NewResponse(&infov1.PeersResponse{
+		NumPeers: 0,
+		Peers:    []*infov1.PeerInfo{},
+	}), nil
+}
+
+func (*stubInfoService) IsBootstrapped(
+	_ context.Context,
+	_ *connect.Request[infov1.IsBootstrappedRequest],
+) (*connect.Response[infov1.IsBootstrappedResponse], error) {
+	return connect.NewResponse(&infov1.IsBootstrappedResponse{
+		IsBootstrapped: true,
+	}), nil
+}
+
+func (*stubInfoService) Upgrades(
+	_ context.Context,
+	_ *connect.Request[infov1.UpgradesRequest],
+) (*connect.Response[infov1.UpgradesResponse], error) {
+	return connect.NewResponse(&infov1.UpgradesResponse{}), nil
+}
+
+func (*stubInfoService) Uptime(
+	_ context.Context,
+	_ *connect.Request[infov1.UptimeRequest],
+) (*connect.Response[infov1.UptimeResponse], error) {
+	return connect.NewResponse(&infov1.UptimeResponse{}), nil
+}
+
+func (*stubInfoService) GetVMs(
+	_ context.Context,
+	_ *connect.Request[infov1.GetVMsRequest],
+) (*connect.Response[infov1.GetVMsResponse], error) {
+	return connect.NewResponse(&infov1.GetVMsResponse{
+		Vms: map[string]*infov1.VMAliases{
+			"avm": {Aliases: []string{"avm"}},
+		},
+		Fxs: map[string]string{},
+	}), nil
+}
+
+var _ = Describe("InfoService ConnectRPC E2E (integration, no stubs)", func() {
```

Comment:
Here's an example of a test that calls the info api, maybe we could try to see if we can get a similar setup where we setup tmpnet nodes and call their apis by hitting their urls: https://github.com/ava-labs/avalanchego/blob/0554b91f73e40bdab3fb1eefde2093ded0b99b05/tests/e2e/p/interchain_workflow.go#L112